hr_hr.lang

```diff
@@ -1,215 +1,349 @@
+gui.xaero_box_height_limit=Maksimalni razmak Y razina za entitete dok se više ne prikazuju.
+gui.xaero_box_compass_over_wp=Prikaži slova na kompasu (S, I, J, Z) preko putnih točaka na karti.
+gui.xaero_toggle_map_waypoints=Uključi putne točke na karti
+gui.xaero_entity_nametags=Oznake s imenom entiteta
+gui.xaero_minimap_text_align=Položaj teksta informacija
+gui.xaero_center=Centar
+gui.xaero_left=Lijevo
+gui.xaero_right=Desno
+gui.xaero_show_angles=Prikaži kutove gledanja
+gui.xaero_box_entity_nametags=Prikaži imena entiteta ispod točke za entitete koji imaju oznaku s imenom.
+gui.xaero_box_teleport_default_command=Naredba korištena za teleportaciju do putnih točaka ako poslužitelj-specifična naredba nije odabrana u izborniku putnih točaka, Opcije ekranu.
+gui.xaero_waypoints_edit_mode_individually=Jedan po jedan
+gui.xaero_waypoints_edit_mode_all=Sve
+gui.xaero_compass=Kompas
+gui.xaero_cave_maps_depth=Dubina špiljskih karti
+gui.xaero_waypoints_edit_reset=Resetiraj
+gui.xaero_main_entity_as=Glavni entitet kao
+gui.xaero_arrow=Strijela
+gui.xaero_dot=Točka
+gui.xaero_crosshair=Nišan
+gui.xaero_teleport_coordinates_hidden=Nemoguće teleportirati jer bi ti reklo koordinate putne točke. Onemogući "Sakrij koordinate putne točke" kako bi se ponovno slobodno teleportirao.
+gui.xaero_hide_wp_coords=Sakrij koordinate putne točke
+gui.xaero_waypoints_edit_show=Prikaži
+gui.xaero_waypoints_edit_hide=Sakrij
+gui.xaero_teleport_anyway=Svejedno teleportiraj
+gui.xaero_teleport_shows_coordinates=Prikaže ti koordinate!
+gui.xaero_render_all_wp_sets=Prikaži sve setove putnih točaka
+gui.xaero_waypoints_global=Globalno
+gui.xaero_waypoints_local=Lokalno
+gui.xaero_player_arrow_opacity=Neprozirnost strelice
+gui.xaero_box_waypoints_distance=Maksimalna udaljenost prikaza za lokalne putne točke. Ne utječe na globalne putne točke.
+gui.xaero_connect_with_auto=Dodaj vezu podsvijetova
+gui.xaero_disconnect_from_auto=Obriši vezu podsvijetova
+gui.xaero_world_connection_tooltip=Poveži/odkači odabrani podsvijet za/od trenutnog automatskog podsvijeta. Poveži samo podsvijetove koji su iz istog savea svijeta. Na primjer, ako te vanilla Nether portal vodi iz podsvijeta A u podsvijet B, onda ih možeš povezati. Podsvijetovi povezani s tvojim trenutnim su uvijek na vrhu popisa i označeni su sa simbolom *.
+gui.xaero_connect_with_auto_msg=Bi li htijeo stvoriti slijedeću vezu između podsvijetova?
+gui.xaero_disconnect_from_auto_msg=Bi li htijeo obrisati slijedeću vezu između podsvijetova?
+gui.xaero_teleport_not_connected=Pokušavaš se teleportirati do podsvijeta koji nije povezan na trenutni automatski. Ako si siguran da je ovaj podsvijet iz istog pod-poslužitelja/savea svijeta, onda možeš omogućiti teleportaciju dodavajući vezu u izborniku putnih točaka -> Opcije -> Dodaj vezu podsvijetova. Ali, ako povežeš nevezane podsvijetove i teleportiraš se, onda bi se mogao ugušiti u bloku ili umrijeti od štete pada, pa ne radi to.

```

Comment:
Please add the new lines at the end:
gui.xaero_ignore_heightmaps=Ignore Server Heightmaps
gui.xaero_box_ignore_heightmaps=Some servers put custom values in the world heightmaps, which can cause incorrectly rendered maps. Sometimes it can even happen in heavily modded singleplayer worlds. This option should fix such issues at the expense of a bit of performance. Do not use unless you need to. Reenter the world after toggling the option!
gui.xaero_error_loading_properties=Error loading server minimap properties. Please retry.

Reply:
Okay, I will add them in this new version

Reply:
Appreciate it, thank you! Seems perfect.